hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSDirRenameOp.java

```diff
@@ -247,10 +247,8 @@ static RenameResult renameToInt(
       throws IOException {
     String src = srcArg;
     String dst = dstArg;
-    if (NameNode.stateChangeLog.isDebugEnabled()) {
-      NameNode.stateChangeLog.debug("DIR* NameSystem.renameTo: with options -" +
-          " " + src + " to " + dst);
-    }
+    NameNode.stateChangeLog.debug("DIR* NameSystem.renameTo: with options={} {} to {}",
+        Arrays.toString(options), src, dst);
```

Comment:
not sure about this...
Wouldn't it incur overhead because it has to compose a string before determining to log or not?

Reply:
Oh, here is my mistake. @jianghuazhu we need `if (NameNode.stateChangeLog.isDebugEnabled())` here,  because `Arrays.toString(options)` will cost some time to compose a string.

Reply:
Thanks for the tip here, @jojochuang .
I will update soon.